### ISSUE TITLE
Fix date now showing on articles imported from wordpress

### ISSUE THREAD

**Issue by alienlebarge** [alienlebarge/alienlebargech-v3#19]
### Description

All articles that were imported from wordpress back in the days have no date.

There's is not `date` in front matter. The ordering is based on the filename.

### Ask

Fix that

### Additional Info

- https://www.alexsears.com/2019/10/add-date-from-file-name-to-frontmatter/
- https://www.11ty.dev/docs/dates/

**Comment by alienlebarge** [comment]
Fix in abe2396eee741b82552e54be0706a728ee8a9772

**Comment by alienlebarge** [comment]
I have run this script

```
#!/usr/bin/env python3
from os import listdir
from os.path import isfile, join
import sys

only_files = [f for f in listdir(sys.argv[1]) if isfile(join(sys.argv[1], f))]
file_dates = [f[:10] for f in only_files]

for (date, filename) in zip(file_dates, only_files):
  filepath = join(sys.argv[1], filename)
  contents = list()
  with open(filepath, 'r') as fd:
    contents = fd.readlines()

  contents = [contents[0]] + [f'date: {date}\n'] + contents[1:]

  with open(filepath, 'w') as fd:
    fd.writelines(contents)
```

But make sure you do not have a `.DS_store` file in the folder.
